.vsts-dotnet-ci.yml

```diff
@@ -63,21 +48,37 @@ phases:
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: MacOS
+    name: MacOS_x64_Netcoreapp21
     buildScript: ./build.sh
     queue:
       name: Hosted macOS
 
 - template: /build/ci/phase-template.yml
   parameters:
-    name: Windows_NetFx
+    name: Windows_x64_Netcoreapp30
+    buildScript: build.cmd
+    customMatrixes:
+      Debug:
+        _configuration: Debug-Intrinsics
+        _config_short: DI
+        _includeBenchmarkData: false
+      Release:
+        _configuration: Release-Intrinsics
+        _config_short: RI
+        _includeBenchmarkData: true
+    queue:
+      name: Hosted VS2017
+
+- template: /build/ci/phase-template.yml
+  parameters:
+    name: Windows_x64_Netfx
     buildScript: build.cmd
     customMatrixes:
-      Build_Debug_netfx:
+      Debug:
         _configuration: Debug-netfx
         _config_short: DFX
         _includeBenchmarkData: false
-      Build_Release_netfx:
+      Release:
```

Comment:
I'd suggest not naming any of these matrix legs anything as simple as "Release" or "Debug".  In the past, we've seen issues where if the Job name (agent variable) is "Release", then some tasks (some MicroBuild task) thinks it is running in a release pipeline and exhibits odd behavior.

Reply:
Wont the job name here will Windows_x64_Netfx Debug ?